src/status_im/contexts/communities/events.cljs

```diff
@@ -478,3 +508,28 @@
 (rf/reg-event-fx :communities/update-last-opened-at-success
  (fn [{:keys [db]} [community-id last-opened-at]]
    {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)}))
+
+(rf/defn share-community
+  {:events [::share-community-confirmation-pressed]}
+  [cofx users community-id]
```

Comment:
The name `users` is a bit confusing. It's not well named in status-go too. I checked the code. The endpoint expects a collection of public keys. So clearer would be to rename `users` to `users-pub-keys` or something along these lines. Just `users` made me think if this wasn't a bug because we normally don't pass large data structures to status-go and prefer to pass only IDs, keys, etc.

Reply:
Renamed